server/routes/superheroes_routes.js

```diff
@@ -5,91 +5,103 @@ const { Superhero } = require('../db/models/models_index.js');
 
 // get all superheroes
 superheroesRouter.get('/', async (req, res) => {
-    
-    try {
-        const superheroes = await Superhero.findAll();
-        res.status(200).send({ superheroes });
-    } catch(e) {
-        console.error(e);
-        res.status(404).send({ message: 'Server error' });
-    }
+  try {
+    const superheroes = await Superhero.findAll();
+
+    res.status(200).send(superheroes);
```

Comment:
Does this need to be 

if(superheroes) { 
  res.status(200).send(superheroes);
} else { 
  res.status(404).send({ message: `Superheroes not found` });
}

Reply:
It will return an empty array if nothing found.